src/main/java/seedu/address/logic/parser/Parser.java

```diff
@@ -37,226 +37,283 @@
  */
 public class Parser {
 
-    /**
-     * Used for initial separation of command word and args.
-     */
-    private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
+	/**
+	 * Used for initial separation of command word and args.
+	 */
```

Comment:
<img width="398" alt="why" src="https://cloud.githubusercontent.com/assets/19604232/19395210/11c86e50-926f-11e6-9c9f-26bd1646f066.PNG">

Multiple markers at this line
	- File contains tab characters (this is the first instance).
	- 'method def modifier' have incorrect indentation level 8, expected level 
	 should be 4.

I dont understand the problem, how to ix this? it looks the same as code which is okay